.travis.yml

```diff
@@ -1,3 +1,7 @@
+branches:
+  only:
+  - master
+  - /^v\d+\.\d+\.\d+$/
```

Comment:
This is a regular expression that matches the git tag names that we use(i.e. v.0.0.3). This ensures that Travis CI will builds GitHub releases.

Reply:
This is explained in more detail in the Travis CI docs. There is a link to the docs in the commit message.